ocdsdata-status

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import argparse
+import os
+from ocdsdata.status import SourceStatus
+
+
+def main():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--source", help="Which source do you want to see status for?")
+    parser.add_argument("--dataversion", help="Specify a data version - defaults to latest one")
+    parser.add_argument("--sample", help="See status for a sample", action="store_true")
```

Comment:
Why do we need to set `action="store_true"` here?

Reply:
Turns it into a boolean. It's in main cli script to.